### PR TITLE
Fix GPT-OSS `swiglu_limit` not passed in for MXFP4

### PR DESCRIPTION
**Pull request by danielhanchen**
`swiglu_limit = 7.0` was not passed in when doing MXFP4 inference when using OpenAI's triton_kernels library.

### CODE REVIEW

**Comment by Rocketknight1** [comment]
cc @arthurzucker or possibly @MekkCyber

**Review by ArthurZucker — APPROVED** [review]
Damn that's a great catch.... let me run the tests on the PR but def merging

**Comment by ArthurZucker** [comment]
run-slow: mxfp4, gpt_oss

**Comment by ArthurZucker** [comment]
Well, the testing suit is fucked up cc @MekkCyber or @SunMarc for when you come back, logits will be different but merging!
